core/slice.go

```diff
@@ -234,23 +223,19 @@ func (sl *Slice) ConstructLocalBlock(header *types.Header) *types.Block {
 }
 
 // updateCacheAndRelay updates the pending headers cache and sends pending headers to subordinates
-func (sl *Slice) updateCacheAndRelay(pendingHeader types.PendingHeader, location common.Location, reorg bool, isCoincident bool) {
+func (sl *Slice) updateCacheAndRelay(pendingHeader types.PendingHeader, reorg bool, isCoincident bool) {
 	nodeCtx := common.NodeLocation.Context()
 
 	sl.phCachemu.Lock()
 	defer sl.phCachemu.Unlock()
 
 	sl.updatePhCache(pendingHeader)
 	if !isCoincident {
-		switch nodeCtx {
-		case common.PRIME_CTX:
-			sl.updatePhCacheFromDom(pendingHeader, 3, []int{common.REGION_CTX, common.ZONE_CTX}, reorg)
-		case common.REGION_CTX:
-			sl.updatePhCacheFromDom(pendingHeader, 3, []int{common.ZONE_CTX}, reorg)
-		case common.ZONE_CTX:
+		if nodeCtx == common.ZONE_CTX {
 			sl.miner.worker.pendingHeaderFeed.Send(sl.phCache[sl.pendingHeader].Header)
 			return
 		}
+		sl.updatePhCacheFromDom(pendingHeader, 3, []int{}, reorg)
```

Comment:
What is magic number 3 here? Can we use a named constant for the humans reading the code?

Reply:
THanks, using terminiIndex now.